PyBitTorrent/Configuration.py

```diff
@@ -0,0 +1,45 @@
+# Port to announce to tracker 
+# port section at: 
+#   https://wiki.theory.org/BitTorrent_Tracker_Protocol#Basic_Tracker_Announce_Request)
+LISTENING_PORT = 6881
```

Comment:
The idea of using Configuration file is GREAT.
although, i would rather it to be in some type of a file:
yaml, json, toml, etc.
this way it would be much more clear for user to change the config.
I dont think it will be to hard to replace this const file with configuration file, and add a class/function that read this config file and represent its value.
can you convert that Configuration.py to that?
i would recommend use Yaml or Json.